{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -6,25 +6,15 @@ skipsdist = True
 
 [testenv]
 setenv =
-    DJANGO_SETTINGS_MODULE={{ cookiecutter.project_slug }}.settings.base
+    DJANGO_SETTINGS_MODULE = {{ cookiecutter.project_slug }}.config.settings.test
     PYTHONPATH={toxinidir}
-    SECRET_KEY=test
     DATABASE_URL=postgresql://{{ cookiecutter.project_slug }}:{{ cookiecutter.project_slug }}@localhost/{{ cookiecutter.project_slug }}
-deps = -r{toxinidir}/requirements/base.txt
+deps = -r{toxinidir}/requirements/dev.txt
```

Comment:
I don't think we should install dev dependencies in the test environment. I would prefer to create a `test` requirements file and install this one.